Userstore manager reference table added. (#3335)
wso2/docs-is: en/docs/deploy/migrate/migrate-to-600.md

If you have created secondary user stores in the previous WSO2 IS version, copy the content in the `<OLD_IS_HOME>/repository/deployment/server/userstores` folder to the `<NEW_IS_HOME>/repository/deployment/server/userstores` folder.

!!! note
    - If you are migrating from a version prior to IS 5.5.0, you need to make the following changes in the `<NEW_IS_HOME>/migration-resources/migration-config.yaml` file (see Step 2 for instructions for downloading migration resources).

>         - Remove all `UserStorePasswordMigrators` from versions above your previous IS version. Userstore password migration will be done by the `EncryptionAdminFlowMigrator` in the version 5.11.0.

Reviewer: ```suggestion
        - Remove all `UserStorePasswordMigrators` from versions above your previous IS version. User store password migration will be done by the `EncryptionAdminFlowMigrator` in version 5.11.0.
```

    
        ```toml
        name: "UserStorePasswordMigrator"
        order: 5
        parameters:
          schema: "identity"
        ```

>         - Change the `currentEncryptionAlgorithm` to `“RSA”` in `EncryptionAdminFlowMigrator` of version 5.11.0

Author: ```suggestion
        - Change the `currentEncryptionAlgorithm` to `RSA` in `EncryptionAdminFlowMigrator` of version 5.11.0
```

    
        ```toml
        name: "EncryptionAdminFlowMigrator"
        order: 1
        parameters:
          currentEncryptionAlgorithm: "RSA"
          migratedEncryptionAlgorithm: "AES/GCM/NoPadding"
          schema: "identity"
        ```

>     - If you are migrating from a version prior to IS 5.10.0, you need to update the UserStoreManager class name in userstore XML files to its respective Unique ID userstore manager class name according to the table below.

Reviewer: ```suggestion
    - If you are migrating from a version prior to IS 5.10.0, you need to update the `UserStoreManager` class name in the userstore XML files to its respective Unique ID userstore manager class name according to the table below.
```
